### ISSUE TITLE
Regarding training code and training settings

### ISSUE THREAD

**Issue by starsong98** [princeton-vl/LayeredFlow#3]
Hello, thank you for releasing your amazing paper, code, pretrained datasets, and pretrained models.

I have a few questions regarding the training procedure though.

1. Do you have any plans to release your training code?
2. Are your implementations of the LayeredFlow dataset classes compatible with the synthetic training split as well?
3. How many steps did you train your models for? I could only find mentions of the learning rate and batch size.
4. What augmentation hyperparameters (i.e., the max and min scales, training image crop sizes) did you apply to the LayeredFlow-train set? Were they identical to those for the Sintel dataset? I was only able to find that S+L mixture was implemented as `100 * sintel_clean + 100 * sintel_final + layeredflow`.

Thank you.
